src/cdn/helpers/opacities.css

```diff
@@ -1,7 +1,19 @@
 .no-opacity {
-  opacity: 0;
+  opacity: 1;
 }
 
 .opacity {
-  opacity: 1;
+  opacity: 0;
+}
+
+.small-opacity {
+  opacity: 0.75;
```

Comment:
IMO we need to avoid !important as possible. It's not a good practice at all and it hurts the specifity of css. But is very useful in some cases. It's ok for me. Nice. (https://github.com/beercss/beercss/commit/9e0e719adab0c2e7910edc2cb228f9d5fd67b539)